test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -523,6 +561,8 @@ private List<Symbol> multiColumnQueryColumns()
             List<Symbol> allowedColumns = searchableColumns;
             if (hasMultiNodeMultiColumnAllowFilteringWithLocalWritesIssue())
                 allowedColumns = nonPkIndexedColumns;
+            if (IGNORED_ISSUES.contains(KnownIssue.SAI_AND_VECTOR_COLUMNS) && !indexes.isEmpty())
```

Comment:
added to get clean CI.  Something is up with multi column queries that touch both SAI and `vector` type.  It doesn't look like the `vector` must be indexed to get incorrect results, so doesn't feel like we are selecting a bad index here